Copyright notice for cmake/FindTBB.cmake:

# Copyright Contributors to the OpenVDB Project
# SPDX-License-Identifier: MPL-2.0

diff -urp openvdb-8.1.0.orig/cmake/FindTBB.cmake openvdb-8.1.0/cmake/FindTBB.cmake
--- openvdb-8.1.0.orig/cmake/FindTBB.cmake	2021-06-28 22:34:03.501173078 -0700
+++ openvdb-8.1.0/cmake/FindTBB.cmake	2021-06-28 22:41:18.794231496 -0700
@@ -3,6 +3,8 @@
 #
 #[=======================================================================[.rst:
 
+# oteodoro - more debug messages to observe favored version which could be legacy TBB or oneTBB
+
 FindTBB
 -------
 
@@ -182,6 +184,7 @@ if(NOT EXISTS ${_tbb_version_file})
   )
   set(_tbb_version_file "${Tbb_INCLUDE_DIR}/oneapi/tbb/version.h")
 endif()
+message(STATUS "TBB include path: ${Tbb_INCLUDE_DIR}")				# oteodoro - added line
 
 if(EXISTS ${_tbb_version_file})
   file(STRINGS ${_tbb_version_file}
@@ -204,6 +207,7 @@ if(EXISTS ${_tbb_version_file})
   unset(_tbb_version_minor_string)
 
   set(Tbb_VERSION ${Tbb_VERSION_MAJOR}.${Tbb_VERSION_MINOR})
+  message(STATUS "TBB version:  ${Tbb_VERSION}")				# oteodoro - added line
 endif()
 
 unset(_tbb_version_file)
@@ -320,6 +324,7 @@ foreach(COMPONENT ${TBB_FIND_COMPONENTS}
   else()
     set(TBB_${COMPONENT}_FOUND FALSE)
   endif()
+  message(STATUS "TBB ${COMPONENT} lib path: ${Tbb_${COMPONENT}_LIBRARY}")				# oteodoro - added line
 endforeach()
 
 # Reset library suffix
